### ISSUE TITLE
“Minimum Distance” field for certain types of orders is missing/invisible/mispositioned

### ISSUE THREAD

**Issue by db48x** [simast/SolarisTheme#1]
The textbox where you enter the minimum distance to follow behind a contact is unavailable, perhaps pushed off the screen because the font is a different size. Is this something that the theme can fix easily?

![Screenshot from 2024-01-19 20-02-58](https://github.com/simast/SolarisTheme/assets/228849/e1523bb0-4a71-4f9e-92a4-be94a6cb8e6b)

**Comment by simast** [comment]
Are you using some kind of font mod or playing this on Linux? This theme should be using Tahoma font (which does not have mentioned issue on my setup):

https://github.com/simast/SolarisTheme/blob/83aa53017754216367851620b4457284db654c15/SolarisTheme.cs#L24-L25

In general, it's not feasible to support different width or size fonts in Aurora. The Windows Forms based UI is just not scalable/flexible.

**Comment by db48x** [comment]
Yep, I figured. I don’t have Tahoma, so I have to substitute something else. I guess technically I could dig it up; I do own an ancient Windows CD and I remember putting it in a box when I moved recently.

Anyway, I asked because I noticed that this theme does actually reposition one or two things to improve the alignment and eliminate overlap. Isn’t it technically possible to measure the sizes of the elements on that line of the UI and reposition them so that they all show up correctly? I’m not sure I have the tools to debug the problem properly. And I suppose that even if it is possible, you wouldn’t want to deal with an endless list of such corrections.
